<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conductor</class>
 <widget class="QDialog" name="Conductor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="conductorID_lineEdit_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Alias:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Section Type:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="conductorSectionTypecomboBox_2">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Overhead</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cable</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="horizontalSpacing">
      <number>23</number>
     </property>
     <item row="0" column="2">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Number Of Phases:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="conductor_R_lineEdit">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Rated Voltage [kV]:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>g  [uS/km]:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="conductor_G_lineEdit">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="conductor_B_lineEdit">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QLineEdit" name="conductorG0_lineEdit">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="3">
      <widget class="QLineEdit" name="conductor_B0_lineEdit">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>b0  [uS/km]:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>x0 [Ohm/km]:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="conductor_X0_lineEdit">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>b  [uS/km]:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>r0  [Ohm/km]:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="conductorNumberOfPhasescomboBox">
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Single Phase</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Two Phases</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Three Phases</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="label_11">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>g0  [uS/km]:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="conductor_R0_lineEdit">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="conductor_X_lineEdit">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>r  [Ohm/km]:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="conductorRatedVoltage_comboBox">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>0.23</string>
       </property>
       <item>
        <property name="text">
         <string>0.23</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.40</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.42</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.52</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6.30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10.38</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10.50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20.50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>33.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>35.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>36.75</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>66.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>110.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>132.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>220.00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>400.00</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>x [Ohm/km]:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Short term rating [A]:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="conductor_mediumrating_lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Medium term rating [A]:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="conductor_longrating_lineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Long term rating [A]:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="conductor_shortrating_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_20">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>Comment:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="conductor_comment_plainTextEdit">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="overwriteMode">
      <bool>false</bool>
     </property>
     <property name="backgroundVisible">
      <bool>false</bool>
     </property>
     <property name="centerOnScroll">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
